ChangeLog

```diff
@@ -78,6 +78,9 @@ Release date: TBA
 
   Closes #1403
 
+* Fixed inference of elements of living container objects such as tuples and sets in the
+  ``sys`` and ``ssl`` modules.
```

Comment:
Does this mean we don't need the imports in the `ssl` brain anymore?
https://github.com/PyCQA/astroid/blob/8e7174a99a9b9dde848632f99b0498f8d67999f3/astroid/brain/brain_ssl.py#L71-L108

Reply:
Sadly it doesn't. Those imports are used to construct `Enums`, which aren't covered by this.

There are some `tuples` in `_ssl` which we do infer now without the brain, but since nobody actually opened an issue about this I don't think anybody ever noticed.

Reply:
👌🏻